lambdas/data-migration2/src/executions.ts

```diff
@@ -97,13 +102,15 @@ export const migrateExecutions = async (
         migrationSummary.skipped += 1;
       } else {
         migrationSummary.failed += 1;
-        logger.error(
-          `Could not create execution record in RDS for Dynamo execution arn ${record.arn}:`,
-          error
-        );
+        errorMessage = `Could not create execution record in RDS for Dynamo execution arn ${record.arn}:`;
+        errorFile.push(`Error: ${error} ${errorMessage}`);
+        logger.error(errorMessage, error);
       }
     }
 
+    if (bucket) {
```

Comment:
Is there a reason for this logic to be optional?

Reply:
Not any good reason